### PR TITLE
ci-update

### PR DESCRIPTION
**Pull request by cygnus-x-1**
updated runner image and travis pipeline

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,23 +5,22 @@ cache:
...
+  - stage: Build and Test
     dist: trusty
     language: java
     services:
       - docker
     script:
-      - docker pull "${runner_image}"
-      - docker run -v "${TRAVIS_BUILD_DIR}:${TRAVIS_BUILD_DIR}" -v "${HOME}"/.ivy2/cache:/root/.ivy2/cache --workdir "${TRAVIS_BUILD_DIR}" "${runner_image}" .travis/run-tests.sh
+      - docker pull "${RUNNER_IMAGE}"
+      - docker run --mount "type=bind,source=${TRAVIS_BUILD_DIR},target=${TRAVIS_BUILD_DIR}" --mount "type=bind,source=${HOME}/.ivy2/cache,target=/root/.ivy2/cache" --workdir "${TRAVIS_BUILD_DIR}" "${RUNNER_IMAGE}" .travis/run-tests.sh
```
Comment (arcturus2) [inline .travis.yml:23]: The `-v` has worked fine so what's the advantage of using `--mount`?

File: nbproject/project.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#Mon, 11 May 2020 08:05:56 +0930
```
Comment (arcturus2) [inline nbproject/project.properties:1]: lol

File: docker/netbeans-runner/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,64 @@
+FROM ubuntu:18.04
```
Comment (arcturus2) [inline docker/netbeans-runner/Dockerfile:1]: is this file meant to replace the other file `Dockerfile.netbeans-11.3` ?

File: .travis.yml (hunk 1 above)
Comment (cygnus-x-1) [reply .travis.yml:23]: --mount is the newer (supposedly more stable) version

File: docker/netbeans-runner/Dockerfile (hunk 3 above)
Comment (cygnus-x-1) [reply docker/netbeans-runner/Dockerfile:1]: nope - i wasn't meant to commit this :\

File: nbproject/project.properties (hunk 2 above)
Comment (cygnus-x-1) [reply nbproject/project.properties:1]: well that didn't last long

**Comment by arcturus2** [comment]
I've just noticed the following warning appear on existing PR's

> warning The version of Java (1.8.0_151) you have used to run this analysis is deprecated and we will stop accepting it from October 2020. Please update to at least Java 11.
Read more here

The link is https://sonarcloud.io/documentation/upcoming/

I am not able to figure out where the `sonar-scanner` is installed by looking at the Docker image. We are definitely using Java 11 with Azul Zulu but it may not be on the class path for the `sonar-scanner` so it falls back to Java 8 some how? The docker image ins't installing Java via `apt-get` so I'm actually not sure how Java 8 is picked up.

@cygnus-x-1 given you've updated versions I was wondering if we could include this fix too?

@TheDen I'd be grateful for any advice on how you setup the `sonar-scanner`.

**Comment by arcturus2** [comment]
@cygnus-x-1 I created https://issues.apache.org/jira/browse/NETBEANS-4418 with the information you sent me.

**Comment by cygnus-x-1** [comment]
This is now ready for review.

**Pull request merged**